### DISCUSSION TITLE
Would be Great if Recurring Could Have Save Features as Outlook

### DISCUSSION THREAD

**Discussion by rwinkel2000** [davish/obsidian-full-calendar#502] [Feature Requests]
### ⚠️ Please check that this feature request hasn't been suggested before.

- [X] I searched previous [Ideas in Discussions](https://github.com/davish/obsidian-full-calendar/discussions/categories/feature-requests) didn't find any similar feature requests.
- [X] I searched previous [Issues](https://github.com/davish/obsidian-full-calendar/issues) didn't find any similar feature requests.

### 🔖 Feature description

Outlook calendar allows several recurrence patterns that are useful: on Saturday but every *two* weeks et al.

### ✔️ Solution

I'd like to be able to create a calendar event that recurs every two weeks instead  of every week.

### ❓ Alternatives

I have considered setting the weekly recurrence and then just remembering that this is not the week to do that thing.

### 📝 Additional Context

Just imitating the outlook calendar Appointment Recurrence pop-up would cover all the bases.

**Comment by bskeen** [comment]
Maybe reference this discussion: #344, we are talking there about enhancements to the recurring events in the plugin.

I also just created a PR the other day that does implement rules like the one you describe: #500. In the above discussion I included screenshots of the updated ones, and it is similar to Outlook in some ways.

Maybe take a look and let me know if there is something additional it would need, so I can make my PR even better.
